allennlp/models/semantic_parsing/wikitables/wikitables_decoder_step.py

```diff
@@ -84,6 +84,7 @@ def take_step(self,
         memory_cell = torch.stack([rnn_state.memory_cell for rnn_state in state.rnn_state])
         previous_action_embedding = torch.stack([rnn_state.previous_action_embedding
                                                  for rnn_state in state.rnn_state])
+        score_so_far = torch.stack(state.score)
```

Comment:
Show the shape of this tensor and may be write a comment describing what the addition below is doing?